manim/mobject/geometry/polygram.py

```diff
@@ -133,18 +134,52 @@ def get_vertex_groups(self) -> np.ndarray:
 
         return np.array(vertex_groups)
 
-    def round_corners(self, radius: float = 0.5):
+    def round_corners(
+        self,
+        radius: float | list[float] = 0.5,
+        evenly_distribute_anchors: bool = False,
+        components_per_rounded_corner: int = 2,
+    ):
         """Rounds off the corners of the :class:`Polygram`.
 
         Parameters
         ----------
-        radius
+        radius : float | list[float]
             The curvature of the corners of the :class:`Polygram`.
+        evenly_distribute_anchors : bool
+            Break long line segments into proportionally-sized segments.
+        components_per_rounded_corner : int
```

Comment:
Type hints added to the methods don't need to be repeated in the docstring.

```suggestion
        radius
            The curvature of the corners of the :class:`Polygram`.
        evenly_distribute_anchors
            Break long line segments into proportionally-sized segments.
        components_per_rounded_corner
```

Reply:
Applied!